manifests/kustomize/third-party/mysql/base/mysql-deployment.yaml

```diff
@@ -20,13 +20,12 @@ spec:
       # https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_ignore-db-dir
       # Ext4, Btrfs etc. volumes root directories have a lost+found directory that should not be treated as a database.
       - args:
-        - --ignore-db-dir=lost+found
         - --datadir
         - /var/lib/mysql
         env:
         - name: MYSQL_ALLOW_EMPTY_PASSWORD
           value: "true"
-        image: gcr.io/ml-pipeline/mysql:5.7
+        image: docker.io/mysql:8.0
```

Comment:
Can you post a link to its Dockerfile?
We need to make sure it is compliant with licenses of Its dependencies and transitive dependencies.

Reply:
missed to send this yeasterday. [dockerhub](https://hub.docker.com/layers/mysql/library/mysql/8.0.25/images/sha256-c1afde725e2cfb627136a299b4d4bd35ae30a31fae1297dd2b3c3c951d9c7240?context=explore), github: https://github.com/docker-library/mysql/blob/a09a716a88ce34a17e16df3f8b091066d90e6f34/8.0/Dockerfile.debian